plugin/src/main/kotlin/io/github/ttypic/swiftklib/gradle/templates/CreatePackageSwift.kt

```diff
@@ -2,61 +2,23 @@ package io.github.ttypic.swiftklib.gradle.templates
 
 import io.github.ttypic.swiftklib.gradle.SwiftPackageDependency
 
-internal fun createPackageSwiftContents(
-    cinteropName: String,
-    dependencies: Collection<SwiftPackageDependency>
-): String = """
-    // swift-tools-version:5.6
-    import PackageDescription
-
-    let package = Package(
-        name: "$cinteropName",
-        products: [
-            .library(
-                name: "$cinteropName",
-                type: .static,
-                targets: ["$cinteropName"])
-        ],
-        dependencies: [
-            ${dependencies.joinToString(",\n            ") { it.toSwiftPackageDeclaration() }}
-        ],
-        targets: [
-            .target(
-                name: "$cinteropName",
-                dependencies: [
-                    ${dependencies.joinToString(",\n                    ") { "\"${it.name}\"" }}
-                ],
-                path: "$cinteropName")
-        ]
-    )
-""".trimIndent()
-
-
-internal fun SwiftPackageDependency.toSwiftPackageDeclaration(): String = when (this) {
+internal fun SwiftPackageDependency.toSwiftArgs(): List<String> = when (this) {
     is SwiftPackageDependency.Local ->
-        """
-        .package(path: "${path.absolutePath}")
-        """.trimIndent()
-
+       emptyList()
     is SwiftPackageDependency.Remote.ExactVersion ->
-        """
-        .package(url: "$url", exact: "$version")
-        """.trimIndent()
-
+        listOf(url, "--exact", version)
     is SwiftPackageDependency.Remote.VersionRange -> {
-        val operator = if (inclusive) "..." else "..<"
-        """
-        .package(url: "$url", "$from"$operator"$to")
-        """.trimIndent()
+        if (inclusive) {
+            // can't do inclusive range from command line
```

Comment:
we can't do inclusive range from command line
but I think it's better to use up-to-next-minor-from.
need some rework here

Reply:
command usage 
`swift package add-dependency <dependency> [--exact <exact>] [--revision <revision>] [--branch <branch>] [--from <from>] [--up-to-next-minor-from <up-to-next-minor-from>] [--to <to>]`